packager/media/codecs/ac4_audio_util.cc

```diff
@@ -0,0 +1,528 @@
+// Copyright 2020 Google Inc. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file or at
+// https://developers.google.com/open-source/licenses/bsd
+
+#include "packager/media/codecs/ac4_audio_util.h"
+
+#include "packager/base/macros.h"
+#include "packager/base/strings/string_number_conversions.h"
+#include "packager/media/base/bit_reader.h"
+#include "packager/media/base/rcheck.h"
+
+namespace shaka {
+namespace media {
+
+namespace {
+
+// Speaker group index
+// Bit,      Location
+// 0(LSB),   Left/Right pair
+// 1,        Centre
+// 2,        Left surround/Right surround pair
+// 3,        Left back/Right back pair
+// 4,        Top front left/Top front right pair
+// 5,        Top back left/Top back right pair
+// 6,        LFE
+// 7,        Top left/Top right pair
+// 8,        Top side left/Top side right pair
+// 9,        Top front centre
+// 10,       Top back centre
+// 11,       Top centre
+// 12,       LFE2
+// 13,       Bottom front left/Bottom front right pair
+// 14,       Bottom front centre
+// 15,       Back centre
+// 16,       Left screen/Right screen pair
+// 17,       Left wide/Right wide pair
+// 18,       Vertical height left/Vertical height right pair
+enum kAC4AudioChannelGroupIndex {
+    kLRpair = 0x1,
```

Comment:
kLRPair to be consistent with the naming below.

Reply:
You are right. Fixed.